src/Lesson7/Square.java

```diff
@@ -0,0 +1,14 @@
+package Lesson7;
+
+public class Square extends Rectangle {
+
+    public double side;
```

Comment:
We have 2 parameters from parent class.
Why don't we use one of them?
Also, why it's puublic?